src/views/OverviewPage.vue

```diff
@@ -55,6 +55,51 @@
         </section>
       </div>
     </div>
+
+    <div class="column">
+      <div class="card">
+        <h1 style="margin-left: 10px;">Recent Notes</h1>
+        <section class="fieldsModified">
+          <div class="cardModified" v-for="note in recentNotes" :key="note.id">
```

Comment:
See if you can make each note into a reusable component (shared by this page and the “Notes” page).

Reply:
fixed